### PR TITLE
SQL Validation: Add stricter checking for the TRIGGER statement

### PR DESCRIPTION
**Pull request by pandah3**
## Description

Adding stricter SQL validation for the `TRIGGER` statement to avoid false positives.

Before, we were checking for:
`TRIGGER` (case sensitive)

This PR:
`CREATE TRIGGER` (case sensitive)
or
```
CREATE DEFINER=`root`@`host` TRIGGER`
```
(case sensitive)

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-import-validate-sql.js <sql file>`
1. Should only catch instances stated above

Tried testing w/:

```
CREATE TRIGGER
CREATE DEFINER=`root`@`%` TRIGGER
CREATE DEFINER=`root`@`host` TRIGGER
CREATE DEFINER=`root`@`127.0.0.1` TRIGGER
CREATE DEFINER=`youruser`@`host` TRIGGER
```

Please let me know if there's any instances I missed! Thank you!

### CODE REVIEW

File: src/lib/validations/sql.js [hunk 1]
```diff
@@ -156,7 +156,8 @@ const checks: Checks = {
 		recommendation: 'Remove these lines',
 	},
 	trigger: {
-		matcher: /TRIGGER/,
+		// Match `CREATE (DEFINER=`root`@`host`) TRIGGER`
+		matcher: /^CREATE (DEFINER=`?(\w*)(`@`)?(\w*\.*%?)*`?)?(| )TRIGGER/,
```
Comment (jblz) [inline src/lib/validations/sql.js]: Now that it's more defined, we can probably make this case-insensitive.  Especially if we ship #681 alongside.

Comment (jblz) [reply src/lib/validations/sql.js]: This doesn't seem to match the string:

```
CREATE (DEFINER=`root`@`host.fdsa.com`) TRIGGER
```

...unless I'm missing something.
https://regex101.com/r/9pCSWm/1

Comment (pandah3) [reply src/lib/validations/sql.js]: Oops, I was missing the optional parentheses around the definer. Fixed the regex to be case-insensitive + match the example string 🙇‍♀️

**Comment by pandah3** [comment]
This catches any `TRIGGER` SQL statements, but it doesn't catch any matching DB values if for some reason the statement was defined as a value (i.e:
```
INSERT INTO `table` VALUES (1, CREATE DEFINER=`root`@`host` TRIGGER`, 'exampleValue)
```

**Review by jblz — APPROVED** [review]
lgtm.  Thanks.

**Pull request merged**